src/utils/datatool/Mp3dInstanceMeshData.cpp

```diff
@@ -84,10 +86,10 @@ bool Mp3dInstanceMeshData::loadMp3dPLY(const std::string& plyFile) {
   perFaceIdxs_.reserve(nFace);
 
   for (int i = 0; i < nVertex; ++i) {
-    vec3f position;
-    vec3f normal;
-    vec2f texCoords;
-    vec3uc rgb;
+    Mn::Vector3 position;
+    Mn::Vector3 normal;
+    Mn::Vector2 texCoords;
+    Mn::Vector3ui rgb;
```

Comment:
Probably meant `Vector3ub` instead, or `Color3ub`? 8 bits per component, not 32.

Too bad this datatool thing has no test coverage at all.